config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade.yaml

```diff
@@ -53,7 +53,7 @@ periodics:
     preset-dind-enabled: "true"
     preset-kind-volume-mounts: "true"
   annotations:
-    testgrid-dashboards: sig-cluster-lifecycle-kubeadm, sig-release-1.31-informing
+    testgrid-dashboards: sig-release-1.31-blocking, sig-cluster-lifecycle-kubeadm
```

Comment:
should you just remove the current release ones and leave the older ones as they are today?

Reply:
that's actually not so trivial to do because we use a generator. is that a problem?
EOD we maintain all the releases branches e2e with equal effort.

Reply:
no, it was just curiosity